.travis.yml

```diff
@@ -32,8 +32,8 @@ before_install:
 
 install:
     - docker load < $CACHE_FILE_IMAGE || true
-    - sudo ionice -c 2 -n 0 nice -n -20 docker-compose pull postgres elasticsearch redis selenium-server webserver
-    - docker-compose up -d postgres elasticsearch redis selenium-server php-fpm webserver
+    - sudo ionice -c 2 -n 0 nice -n -20 docker-compose pull postgres elasticsearch redis rabbitmq selenium-server webserver
+    - docker-compose up -d postgres elasticsearch redis selenium-server rabbitmq php-fpm webserver
```

Comment:
It's exactly like you said. Currently, RabbitMQ is not used in tests. Do you think the consumer should be added even when it's not used?

Reply:
I'm not sure, to be honest. I have a gut feeling that it would be better to start the consumer up as well, but since the job doesn't fail when the consumer fails to start up, I don't see any real benefit...

Reply:
I keep it as it is and when the need came up (maybe with another consumer) we should add it.